modules/dummy-module-xyz.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-the-profile-creator-tool"]
+= About the Performance Profile Creator
+
+The PPC is a command-line tool, delivered with the Node Tuning Operator, used to create the performance profile.
+The tool consumes `must-gather` data from the cluster and several user-supplied profile arguments. The PPC generates a performance profile that is appropriate for your hardware and topology.
+
+The tool is executed by one of the following methods:
+
+* Invoking `podman`.
```

Comment:
```suggestion
* Invoking `podman`
```

See IBM Style for https://www.ibm.com/docs/en/ibm-style?topic=format-lists#punctuation-in-lists

> Write lists so that all or none of the items start with complete sentences. If list items comprise only complete sentences, include a period after each sentence.